FAQs.md

```diff
@@ -0,0 +1,102 @@
+# NoteCrypt Frequently Asked Questions(FAQs).
+This is a compilation of Frequently Asked Questions for NoteCrypt. This document will aid prospective and existing end-users of the software as a self-service guide in cases where the developer can't be reached. This document is subject to change and modifications at any time. Thus it'll be updated from time to time consequently to suit any new changes to the software as new developments would be implemented which might cause a discrepancy with this current document.
+
+**What is NoteCrypt?**
+- NoteCrypt is an Android software that helps you secure your supposedly private notes in an encrypted database.
+
+**What are the features of NoteCrypt?**
+
+NoteCrypt has quite a handful of features they include;
+
+- Multiple database creation with different passwords.
+- Create a new note.
+- Tag filter
+- Favorite note
+- Note sorting
+- The database is securely encrypted with AES 128 bit, using cipher-block chaining (CBC), PKCS5Padding and a random IV generated at every save. For the KDF a random salt and PBKDF2WithHmacSHA1 are used.
+- Auto create a hidden backup database to prevent data loss in case the file gets corrupted while saving (the backup file is saved with the name: ."database name").
+- Note search.
+
+**What is the minimum length of characters to set a password?**
+
+- The minimum password for a database is one character, however, it's not advisable to secure a database with just a single digit or alphabet. NoteCrypt allows numeric and alphabetic password entry. For more security, create a password mixed with alphabets and numbers up to 8 characters.
```

Comment:
At the end we are talking of ways to achieve more security. So instead of "up to" it should be "at least 8 characters"